src/main/java/unrc/dose/UserStat.java

```diff
@@ -0,0 +1,15 @@
+/* db-attributes of userStats:
```

Comment:
Comments should be moved to the beginning of class creation (i.e. before public class). Also, it would be better if field types are specified